packages/styled-components/src/components/primitives/Dropdown/Dropdown.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Dropdown } from '.';
+import { text } from '@storybook/addon-knobs';
+
+export default {
+  title: 'Primitives/Dropdown',
+  component: Dropdown,
+};
+
+export const Playground = () => (
+  <>
+    <h1>Playground</h1>
+    <Dropdown title={text('Title', 'Title')}>
+      {text('Content', 'Content')}
+    </Dropdown>
+  </>
```

Comment:
can we reproduce all cases we have?

Reply:
Added

Reply:
we have a list of items, items with icons and two columns right now

Reply:
Added